CORRELATION_CONTEXT_HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,73 @@
+# Correlation Context HTTP Header Format
+
+A correlation context header is used to pass the name-value context properties for the trace. This is a companion header for the `Trace-Context`. The values should be passed along to any child requests. Note that uniqueness of the key within the `Correlation-Context` is not guaranteed. Context received from upstream service may be altered before passing it along.
```

Comment:
use cases can go here, or at least one example. For example, one can pass vendor-specific data here which helps resolve a trace identifier against an existing format